### PR TITLE
Deprecate the API_ROOT constant

### PR DESCRIPTION
**Pull request by mdellweg**
It was turned into a setting.

re #2148
[noissue]

### CODE REVIEW

**Review by fao89 — APPROVED** [review]
LGTM!

File: CHANGES/plugin_api/2148.removal [hunk 1]
```diff
@@ -0,0 +1,2 @@
+The ``API_ROOT`` constant has been deprecated and is scheduled for removal in 3.20. Please use the
```
Comment (ipanova) [inline CHANGES/plugin_api/2148.removal:1]: the changelog entry should be then .deprecation and not removal, i think

Comment (ipanova) [reply CHANGES/plugin_api/2148.removal:1]: how about "The ``API_ROOT`` constant has been deprecated and turned into a setting.  Its removal is scheduled for 3.20. Please use the setting instead."

**Comment by bmbouter** [comment]
Excellentay!

**Pull request merged**